scripts/generateMetadata.js

```diff
@@ -18,16 +26,15 @@ function logSuccess(message) {
 }
 
 async function docgen(filePath) {
-  // Return if file is not supported (no js files and test files)
-  if (!filePath.match(/\.(js)$/i) || filePath.match(/\.(test\.js)$/i)) {
+  if (excludedPaths.includes(filePath)) {
     return null;
   }
 
   const contents = await fs.promises.readFile(filePath, 'utf-8');
 
   // Not all files have data to parse
   try {
-    const parsed = reactDocs.parse(contents);
+    const [parsed] = (await reactDocs).parse(contents);
```

Comment:
react-docgen v6 outputs array of objects. Here I am assuming there's only going to be one component in a given file.